ytfzf

```diff
@@ -620,10 +627,14 @@ scrape_channel () {
 	]')
 
 	videos_json=$(printf "%s" "$videos_json" | jq '.[0:'$sub_link_count']')
-	videos_data=$(get_video_data "$videos_json")
+	[ -z "$videos_data" ] && videos_data=$(get_video_data "$videos_json")
```

Comment:
Here, why do we need to check for `videos_data` being empty?

Reply:
`videos_data` gets reset every loop when there is a new search, checking if its empty allows for people to set videos_data in on_get_search or any function set in config without it being overwritten. Doing this allows an implantation of !! searches purely in config